TUTORIAL.md

```diff
@@ -572,29 +584,40 @@ Flow(
 
 
 
-Previous flow was a bit complicated, but luckily we have the `join`, `concatenate` and `filter_rows` processors which make such combinations a snap
+Previous flow was a bit complicated, but luckily we have the `join`, `concatenate` and `filter_rows` processors which make such combinations a snap:
 
 
 ```python
-from dataflows import Flow, load, dump_to_path, join, concatenate, filter_rows, printer, checkpoint
+from dataflows import (
+    Flow,
+    load,
+    dump_to_path,
+    join,
+    concatenate,
+    filter_rows,
+    printer,
+    checkpoint,
```

Comment:
Even for the last items, it is useful to have a comma as the imports can be reordered without trouble.